plugins/modules/site_workflow_manager.py

```diff
@@ -760,6 +1088,47 @@ def get_want(self, config):
             parameters such as 'site_params' and 'site_name.' The gathered
             information is stored in the 'want' attribute for later reference.
         """
+        try:
+            type_info = self.config[0]['site']['type']
+            want_list = []
+
+            if type_info == "bulk":
+                if self.dnac_version >= self.version_2_3_7_6:
+                    self.keymap = {}
+                    self.keymap = self.map_config_key_to_api_param(
+                        self.keymap, config)
+                    self.keymap.update({
+                        "floor_number": "floorNumber",
+                        "rf_model": "rfModel",
+                        "parent_name_hierarchy": "parentNameHierarchy",
+                        "units_of_measure": "unitsOfMeasure"
+                    })
+                    self.log(self.keymap, "INFO")
+                else:
+                    self.log(
+                        "In this version '{0}' Bulk site operation file is not supported.".format(
+                            self.payload.get("dnac_version")), "ERROR"
+                    )
+                    self.module.fail_json(msg="In this version '{0}' Bulk site operation file is not supported.".format(
+                        self.payload.get("dnac_version")))
+
+                for site_detail in self.config[0]['site']['site_details']:
+                    want = {}
+                    for key, value in site_detail.items():
+                        if value is not None:
+                            mapped_key = self.keymap.get(key, key)
+                            want[mapped_key] = value
+                    want_list.append(want)
+
+                self.want = want_list
+                self.log("Desired State (want): {0}".format(
+                    str(self.want)), "INFO")
+                return self
+        except KeyError as e:
```

Comment:
this exception you can handled using '.get()' while accessing the key.

Reply:
Modified